src/components/extensions/donate/hooks/useBuildBeforeMessegeSend.tsx

```diff
@@ -0,0 +1,171 @@
+import { useGetChainDataByNetwork } from '@/services/chainsInfo/query'
```

Comment:
typo in file name Message

So maybe cal it like useBuildDontationMessage ?

Because BeforeMessageSend is unclear for reading